backend/src/core/interfaces/campaign.interface.ts

```diff
@@ -59,6 +59,7 @@ export interface CampaignStatsCount {
   unsent: number
   sent: number
   invalid: number
+  updatedAt?: Date
```

Comment:
Why optional

Reply:
Because the return type here https://github.com/opengovsg/postmangovsg/blob/f65fa61481636350bffdd9bc90108567a1163792/backend/src/core/services/stats.service.ts#L76
doesn't have an `updatedAt`